app/lib/package.rb

```diff
@@ -64,7 +75,7 @@ def write_page_to_file(page)
   end
 
   def add_attachments(zip_file, manifest)
-    @translation.resource.attachments.where(is_zipped: true).each do |a|
+    @translation.resource.attachments.where(file_file_name: @resources.uniq).each do |a|
```

Comment:
Do you truly no longer care about `is_zipped`? Should that attribute go away?

Reply:
This logic is meant to completely replaced the functionality `is_zipped` provided. but leaving it in place for now would allow us to rollback if necessary.